packages/microsite/src/hydrate.tsx

```diff
@@ -5,6 +5,7 @@ const isServer = typeof window === "undefined";
 export const HydrateContext = createContext<string | false>(false);
 
 export interface HydrationProps {
+  displayName?: string;
```

Comment:
Hmm, does this require passing `displayName` manually for named exports? I'd like to avoid that, but if it's all we can do for now that's fine. I wonder if there's another workaround...

```
export const MyNamedExport = withHydrate(Component, { displayName: 'MyNamedExport' })
```

Reply:
Okay I messed around with this for an hour and couldn't come up with a better solution! I think this is good. Does `displayName` make more sense than something like `as` or `exportedAs`?

Reply:
The other potential workaround (which I didn’t try but assume works) is a named `function`, but that’s increasingly uncommon/unidiomatic, and has semantic differences that might trip people up.

I went with `displayName` because it’s the same name used for this purpose in React/Preact so should be moderately familiar.